include/deal.II/fe/fe_values.h

```diff
@@ -292,6 +330,19 @@ namespace FEValuesViews
                               std::vector<typename ProductType<value_type,typename InputVector::value_type>::type> &values) const;
 
     /**
+     * Same as above, but using a vector local degree-of-freedom values.
+     *
+     * The DoF values typically would be obtained in the following way:
+     * <code>
+     * std::vector<Number> local_dof_values(n_dofs_per_cell);
+     * cell->get_dof_values(solution,local_dof_values.begin(), local_dof_values.end());
```

Comment:
add a space after the first comma.

Maybe say that if `Number` is a regular scalar type such as `double` or `float`, then this can more easily be written as
```
  Vector<double> local_dof_values (...);
  cell->get_dof_values (solution, local_solution_values);
```